website/static/css/main.css

```diff
@@ -18,7 +23,7 @@
 }
 .no-shadow {
     box-shadow: 0 0 0 #000;
-}
+} 
```

Comment:
Merging this time, but make sure you do not do mistakes like this again. We seriously need linters!